ossrelease/github_release.py

```diff
@@ -0,0 +1,36 @@
+from github import Github
```

Comment:
Is this from `PyGithub`? Let's document that as a requirement/dependency somewhere.

That import should probably be moved to a separate `# Import 3rd party libs` line too, just to keep things organized. I know that is picky, but might as well start off that way :)